src/prebid.js

```diff
@@ -448,6 +461,8 @@ $$PREBID_GLOBAL$$.requestBids = hook('async', function ({ bidsBackHandler, timeo
 
   utils.logInfo('Invoking $$PREBID_GLOBAL$$.requestBids', arguments);
 
+  adUnits = checkAdUnitSetup(adUnits);
```

Comment:
Moving call to checkAdUnitSetup before we calculate the value of `adUnitCodes`. This is required because if some adUnit is removed in the function ,`checkAdUnitSetup`, then, the code below will incorrectly include the deactivated adUnit's code in the `adUnitCodes` variable.